specs/typography.md

```diff
@@ -0,0 +1,177 @@
+# Typography
+
+*TODO: Sections that will be removed upon review are marked with ~~striketrough~~, remaining for commenting purposes.*
+
+*TODO: Add handling of line height*
+
+## Overview
+
+*Typography* defines the styles and type ramp that can be applied to standard elements. This spec does not currently propose or define any new components.
+
+### Background
+
+Standard html elements like ```<h1>```, ```<p>```, ```<label>```, and ```<span>``` provide options for semantically defining page hierarchy and formatting. Without styles applied, however, these elements will not coordinate properly with the adaptive styling applied to other components.
+
+*Base size*
+
+While the default styling could vary from browser to browser, most seem to adopt the same default styling for compatibility with existing pages. The default base font size is **16px**.
+
+*Heading size*
+
+[Heading element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements) font sizes are calculated from the base size value as follows:
+
+| Element    | Relative | Size   |
+|------------|----------|--------|
+| ```<h1>``` | 2em      | 32px   |
+| ```<h2>``` | 1.5em    | 24px   |
+| ```<h3>``` | 1.17em   | 18.7px |
+| ```<h4>``` | 1em      | 16px   |
+| ```<h5>``` | 0.83em   | 13.2px |
+| ```<h6>``` | 0.67em   | 10.7px |
+
+Any of these elements may be used with FAST-DNA styles applied.
```

Comment:
Can you elaborate on the emphasis with heading here? I'm not sure I understand the overall call out with headings, user agent styling values and how those relate to our typography story.